GameData/RP-0/Tree/ProceduralAvionics.cfg

```diff
@@ -27,6 +27,15 @@
 		}
 	}
 	
+	MODULE {
```

Comment:
Once again you're not following the same style as the other configs in this file.

Reply:
Old habits die hard